atmos_model.F90

```diff
@@ -101,6 +102,9 @@ module atmos_model_mod
                               block_data_copy_or_fill,                   &
                               block_data_combine_fractions
 
+#ifdef MOVING_NEST
+use fv_moving_nest_main_mod, only: update_moving_nest, dump_moving_nest
+#endif MOVING_NEST
```

Comment:
Please remove `MOVING_NEST` after `#endif`. GNU compiler complains about extra token at the end of #ifdef directive. Here and at two other places.